lib/reveal-ck/config.rb

```diff
@@ -50,7 +50,8 @@ def filter_defaults
                       'HTML::Pipeline::MentionFilter',
                       'HTML::Pipeline::AutolinkFilter'],
         'asset_root' => 'https://assets-cdn.github.com/images/icons/',
-        'base_url' => 'https://github.com'
+        'base_url' => 'https://github.com',
+        'requires' => []
```

Comment:
thanks. :octocat: FTW

Huh, wonder why this is working.

Sure enough, when I change to master, this stops working. :confused: